gremlin-javascript/src/main/javascript/gremlin-javascript/test/helper.js

```diff
@@ -48,5 +48,5 @@ exports.getClient = function getClient(traversalSource) {
 };
 
 exports.getSessionClient = function getSessionClient(traversalSource) {
-  return new Client(serverUrl, { 'traversalSource': traversalSource, 'processor': 'session'});
-};
\ No newline at end of file
+  return new Client(serverUrl, { 'traversalSource': traversalSource, 'session': 'unique-string-id'});
```

Comment:
NIT: For the tests, I would avoid using the same identifier for the session as the documentation. It may wrongly imply that `unique-string-id` means something.

```suggestion
  return new Client(serverUrl, { 'traversalSource': traversalSource, 'session': 'my-nodejs-session'});
```

Reply:
ok, it's better to put a uuid-string, but I'm lazy here